cmake/DetectCodeCoverage.cmake

```diff
@@ -20,7 +26,8 @@ if ( GCOV AND
      LCOV AND
```

Comment:
More bg info on the logic: Currently we have three code coverage targets: `unittest_coverage`, `regression_coverage`, and `test_coverage`, where the last one is the coverage for both. (I wanted to see detailed (line-)coverage depending on the type of tests.)